### ISSUE TITLE
Gateway UI: Gatewayd API Integration

### ISSUE THREAD

**Issue by okjodom** [fedimint/fedimint#2130]
Mintgate defines and uses [mock Gatewayd API](https://github.com/fedimint/fedimint/blob/master/gateway/ui/src/api/Mintgate.mock.ts) which is really nice for development without starting a real Gatewayd instance.

Here is [how it selects an API to use](https://github.com/GETLN/mintgate/blob/main/src/index.tsx#L10:L14) based on environment variable.

We need to plug in to [the real Gatewayd API](https://github.com/fedimint/fedimint/blob/master/gateway/ui/src/index.tsx#L10:L14)

**Comment by EthnTuttle** [comment]
Move to UI repo? It's unclear to me without context.

**Comment by justinmoon** [comment]
Yes I think so. With a fixed docker-compose setup the UI should be able to be integrated against that.

**Comment by okjodom** [comment]
Yeap. Moving over
